x/gov/types/v1/params.go

```diff
@@ -27,6 +27,9 @@ var (
 	DefaultMinInitialDepositRatio    = sdk.ZeroDec()
 	DefaultProposalCancelRatio       = sdk.MustNewDecFromStr("0.5")
 	DefaultProposalCancelDestAddress = ""
+	DefaultBurnProposal              = false // set to false to replicate behavior of when this change was made (0.47)
+	DefaultBurnVoteQuorom            = false // set to false to  replicate behavior of when this change was made (0.47)
+	DefaultBurnVoteVeto              = true  // set to true to replicate behavior of when this change was made (0.47)
```

Comment:
Agreeing with this, maybe instead of not burning, the condition should send it to the community pool?

Reply:
This is the existing behavior. Don't want to chains and allow chains to decide.

Reply:
Yea, I meant instead of disabling the burn, change the setting to allow send the deposit to the community pool.
Disabling the burn, as Atheesh mentioned effectively transform a no with veto in a no.

Reply:
we can add another param to decide if its burn or send to community pool? but its up to the chain what they would want to set here. Would rather leave it to chains then we decide for them

Reply:
Okay, I'm thinking this param is not needed, community in any chain may want to punish the proposal depositors if it is not appropriate. Sorry if I'm missing context.